src/components/Navigation.vue

```diff
@@ -1,9 +0,0 @@
-<template>
```

Comment:
this didn't seem like it was being used anywhere and my GitLens extension was telling me that it was added on `first commit` so I'm assuming this is outdated lol